odxtools/loadfile.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: MIT
+import os
+from pathlib import Path
+
+from .database import Database
+
+
+def load_pdx_file(pdx_file: str) -> Database:
+    db = Database()
+    db.add_pdx_file(pdx_file)
+    db.refresh()
+    return db
+
+
+def load_odx_d_file(odx_d_file_name: str) -> Database:
+    db = Database()
+    db.add_odx_file(odx_d_file_name)
+    db.refresh()
+
+    return db
+
+
+def load_file(file_name: str) -> Database:
+    if file_name.lower().endswith(".pdx"):
+        return load_pdx_file(file_name)
+    elif file_name.lower().endswith(".odx-d"):
+        return load_odx_d_file(file_name)
+    else:
+        raise RuntimeError(f"Could not guess the file format of file '{file_name}'!")
+
+
+def load_files(*file_names: str) -> Database:
```

Comment:
Nice to have: allow users to provide a Path object or str

Reply:
fixed: https://github.com/mercedes-benz/odxtools/pull/307/commits/c008db2d5ceaf2e091643ffbe0eb22ff720aa967